test/runners/java_spec.js

```diff
@@ -7,13 +7,14 @@ describe('java runner', function() {
     it('should handle basic code evaluation', function(done) {
       runner.run({
         language: 'java',
-        code: [
-          'class Solution {',
-          '   static void main(String[] args){',
-          '        System.out.println("42");',
-          '   }',
-          '}'].join('\n')
+        code: [`
+          class Solution {
+             public static void main(String[] args){
+                  System.out.println("42");
+             }
+          }`].join('\n')
```

Comment:
Minor thing, but this can drop the `[` and `].join('\n')` if we're using template strings.